examples/unwrap_script.py

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/python3
+# -*- coding: utf-8 -*-
+""" Description
+
+"""
+
+
+from shimmingtoolbox.unwrap import unwrap_phase
+import nibabel as nib
+import numpy as np
+import matplotlib.pyplot as plt
+import cmath
+
+
+def main():
+
+    # Import phase
+    phasePath = ""
```

Comment:
we use snake_case for variables, as per [our conventions](https://github.com/shimming-toolbox/shimming-toolbox-py/wiki)